Applications/Custom/LayerClient/jni/Android.mk

```diff
@@ -0,0 +1,70 @@
+LOCAL_PATH := $(call my-dir)
+
+include $(CLEAR_VARS)
+
+# ndk path
+ifndef ANDROID_NDK
+$(error ANDROID_NDK is not defined!)
+endif
+
+ifndef NNTRAINER_ROOT
+NNTRAINER_ROOT := $(LOCAL_PATH)/../../../..
+endif
+
+NNTRAINER_INCLUDES := $(NNTRAINER_ROOT)/nntrainer \
+	$(NNTRAINER_ROOT)/nntrainer/dataset \
```

Comment:
is this intended? client wont have access to these headers

Reply:
If for now, not exposed to API, then this is fine.

Reply:
Yes, I'll add todo here as well (in 779)